packages/commcare/test/index.js

```diff
@@ -319,6 +319,54 @@ describe('getCases', () => {
       },
     ]);
   });
+
+  it('should fetch cases with pagination', async () => {
```

Comment:
I would like this test to also check that the callback is called three times, once for each item